utils/multisig.ts

```diff
@@ -85,6 +85,9 @@ export class Multisig {
     args: any[],
     signers: ethers.Wallet[]
   ): Promise<ethers.types.Transaction> {
+    if (toContract.interface.functions[funcName] === undefined) {
+      throw new Error(`Tried to execute delegateCall to ${funcName} but no such function exists on the target contract`)
```

Comment:
The docstring & this comment should be consistent in using the word call vs delegatecall.